src/OrbitGl/TimelineTicks.cpp

```diff
@@ -70,6 +71,11 @@ std::vector<uint64_t> TimelineTicks::GetMajorTicks(uint64_t start_ns, uint64_t e
   return major_ticks;
 }
 
+uint64_t TimelineTicks::GetPreviousScale(uint64_t tick_scale) const {
```

Comment:
I like Florian suggestion and I will rename it to GetMajorTicksScale and GetMinorTicksScale. In this way, it can be easier to understand and modify later. What do you think?

Reply:
SGTM!